openpype/hosts/nuke/startup/custom_write_node.py

```diff
@@ -53,24 +58,74 @@
 }
 
 
+class WriteNodeKnobSettingPanel(nukescripts.PythonPanel):
+    """ Write Node's Knobs Settings Panel """
+    def __init__(self):
+        nukescripts.PythonPanel.__init__(self, "Set Knobs Value(Write Node)")
+
+        knobs_value, _ = self.get_node_knobs_setting()
+        # create knobs
+
+        self.typeKnob = nuke.Enumeration_Knob(
+            'override_subsets', 'override subsets', knobs_value)
+        # add knobs to panel
+        self.addKnob(self.typeKnob)
+
+    def process(self):
+        """ Process the panel values. """
+        write_selected_nodes = [
+            s for s in nuke.selectedNodes() if s.Class() == "Write"]
+
+        node_knobs = self.typeKnob.value()
+        ext = None
+        knobs = knobs_setting["knobs"]
+
+        if node_knobs:
+            _, node_knobs_settings = self.get_node_knobs_setting(node_knobs)
```

Comment:
Please make sure this will only return `Write` class presets. I am having two presets with the same subset name one for Write and another for Reformat and it is picking up the Reformat and fails to do the job as expected.